lib/licensee/content_helper.rb

```diff
@@ -12,6 +12,18 @@ module ContentHelper
     MARKDOWN_HEADING_REGEX = /\A\s*#+/
     VERSION_REGEX = /\Aversion.*$/i
     MARKUP_REGEX = /[#_*=~\[\]()`|>]+/
+    CC_LEGAL_CODE_REGEX = /^\s*Creative Commons Legal Code\s*$/i
+    CC0_INFO = 'For more information, please see\s*' +
+               '<http://creativecommons.org/publicdomain/zero/1.0/>\s*'.freeze
+    CC0_INFO_REGEX = /#{CC0_INFO}/im
+    CC_DISCLAIMER = %(CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES
```

Comment:
You may want to label this `CC0_DISCLAIMER` or some such.  The other CC licenses have similar disclaimers, but adds additional wording for “information”.  For example, compare the [CC-BY-3.0][1] disclaimer.  The [CC-BY-4.0][2] disclaimer is even more different, dropping caps, adding `("Creative Commons")`, etc.

[1]: https://creativecommons.org/licenses/by/3.0/legalcode.txt
[2]: https://creativecommons.org/licenses/by/4.0/legalcode.txt